### PR TITLE
Error reporting log ssr fixes

### PR DESCRIPTION
**Pull request by alabiaga**
address @choumx comments from #13867

### CODE REVIEW

File: src/error.js [hunk 1]
```diff
@@ -376,13 +376,14 @@ export function maybeReportErrorToViewer(win, data) {
...
-    's': errorReportData['s'], // error stack
-    'el': errorReportData['el'], // tagName
-    'v': errorReportData['v'], // runtime
-    'jse': errorReportData['jse'], // detectedJsEngine
+    m, // message
+    a, // isUserError
+    s, // error stack
+    el, // tagName
+    v, // runtime
+    jse, // detectedJsEngine
```
Comment (choumx) [inline src/error.js]: Does this avoid the binary size increase in the original PR?

Comment (alabiaga) [reply src/error.js]: still need to make the comparison. Will update later. Thanks for the input.

Comment (alabiaga) [reply src/error.js]: w/o change: PASS  ./dist/v0.js: 81.21KB < maxSize 81.26KB (gzip)
with change: PASS  ./dist/v0.js: 81.22KB < maxSize 81.26KB (gzip)

ran 
>> gulp dist --fortesting --noextensions && gulp bundle-size

So there are no savings. This reads better, so let's keep.

Comment (choumx) [reply src/error.js]: I think you'll still need string keys without an extern. Let's just revert this and keep the test changes. 

You still need to recreate this PR to merge to the correct amphtml branch.

Comment (alabiaga) [reply src/error.js]: acknowledged. Thanks

**Pull request closed without merging**